core/src/dunit/scala/io/snappydata/cluster/SplitClusterDUnitTest.scala

```diff
@@ -688,31 +689,20 @@ object SplitClusterDUnitTest extends SplitClusterDUnitTestObject {
     val hostName = InetAddress.getLocalHost.getHostName
     val sparkShellCommand = s"$sparkProductDir/bin/spark-shell --master spark://$hostName:7077" +
         " --conf spark.snappydata.connection=localhost:" + locatorClientPort +
-        s" --jars $snappyDataCoreJar" +
-        securityConf +
-        s" -i $scriptFile -i $scriptFile2"
-
-    logInfo(s"Invoking spark-shell: $sparkShellCommand")
-    val conn = getConnection(locatorClientPort, props)
-    val stmt = conn.createStatement()
-
-    runSparkShellSnappyPoolTest(stmt, sparkShellCommand)
-
-    // accessing tables created through spark-shell
-    val rs1 = stmt.executeQuery("select count(*) from coltable")
-    rs1.next()
-    assert(rs1.getInt(1) == 5)
+        s" --jars $snappyDataCoreJar" + securityConf
```

Comment:
Not clear why all these changes have been done. Don't you just need to add your SQL to the script file as we discussed?